index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="main-content">
+      <nav>
+        <span>Muhyideen Elias</span>
+        <img src="./images/menu.svg" alt="menu" />
+      </nav>
+      <div class="hero">
+        <div class="hero-content">
+          <h3>MUHYIDEEN ELIAS</h3>
+          <h5>Software Developer</h5>
+          <p>
+            I can help you build a product ,<br />
+            feature or website Look through<br />
+            some of my work and experience! If <br />
+            you like what you see and have a<br />
+            project you need coded, don’t <br />
+            hestiate to contact me.
+          </p>
+        </div>
+      </div>
+      <img class="top-circle" src="./images/top-circle.svg" alt="top-circle" />
+      <div class="technologies">
+        <div class="tech-content-1">
+          <div class="languages">
+            <p id="language">Language</p>
+            <p>Javascript</p>
+            <p>Ruby</p>
+            <p>Html</p>
+            <p>Css</p>
+          </div>
+          <div class="frameworks">
+            <p id="framework">Frameworks</p>
+            <p>Boostrap</p>
+            <p>Ruby on Rails</p>
+            <p>RSpec</p>
+            <p>CapyBara</p>
+            <p>Selenium</p>
+          </div>
+        </div>
+        <div class="tech-content-2"></div>
+      </div>
+    </div>
```

Comment:
- [ ] Instead of a `div` tag, this section should be enclosed in a `section`. This is because this block defines who you are and it is semantic this way 👍